QUICKSTART.md

```diff
@@ -142,63 +142,50 @@ In order to avoid that node in your allocation, a simple MSR read test can be
 performed to determine whether or not msr-safe is functional.  In order to
 attempt a read of an MSR through msr-safe, the ```geopmread``` command line
 utility can be utilized.  This must be invoked on the compute nodes through
-```aprun```.  The following bash script can be utilized to determine the
-hostnames of the nodes that are malfunctioning:
-
-#### check_rdmsr.sh
-```bash
-#!/bin/bash
-
-# Try to read a MSR.  Print the hostname if it fails.
-geopmread POWER_PACKAGE_TDP package >& /dev/null || hostname | sed -e 's/nid[0]*//'
```

Comment:
This needs a domain index, i.e.: "geopmread POWER_PACKAGE_TDP package 0"

Reply:
Not sure this is going to work.  When I try this and it fails, there's no way to get rid of the "Illegal Instruction" text:

```$ geopmread POWER_PACKAGE_TDP package 0 >& /dev/null || hostname
Illegal instruction
thetamom3```

This is on a mom node, so the driver isn't installed.  Not sure what the error would be on a node where the driver *IS* installed but misbehaving.  Testing is needed...